cocotb/scheduler.py

```diff
@@ -184,11 +184,11 @@ class Scheduler:
         - Any pending writes are cached and do not happen immediately
 
     ReadOnly mode
-        - Corresponds to :any:`cbReadOnlySynch` (VPI) or :any:`vhpiCbLastKnownDeltaCycle`
+        - Corresponds to :any:`cbReadOnlySynch` (VPI) or :any:`vhpiCbRepEndOfTimeStep`
           (VHPI).  In this state we are not allowed to perform writes.
 
     Write mode
-        - Corresponds to :any:`cbReadWriteSynch` (VPI) or :c:macro:`vhpiCbEndOfProcesses` (VHPI)
+        - Corresponds to :any:`cbReadWriteSynch` (VPI) or :c:macro:`vhpiCbRepLastKnownDeltaCycle` (VHPI)
```

Comment:
The other ones are linked using `:any:` and this one with `:c:macro:`, I don't think either actually link so perhaps it doesn't matter.

Reply:
They actually do link (both): https://docs.cocotb.org/en/latest/library_reference.html#cocotb.scheduler.Scheduler

Reply:
We are documenting the VPI and VHPI in this project?

Reply:
A tiny bit, kind of, due to Doxygen picking up the headers and breathe converting it to Sphinx. Would you prefer not to link because it's mostly useless here?

Reply:
That can probably be handled a little more elegantly. We don't even need the whole generate apidoc and include it, we can directly use doxygen directives. We can list out the files we actually want in the C++ documentation.

Of course there is the issue: what should be in the C++ documentation? None of it is user-face code besides maybe the GPI, which we aren't version controlling rn. It does seem like the best place to put the C++ documentation... Maybe we should unlist it in the toctree? But still be able to access it for linking to/from the internal docs? Just throwing stuff out there...

Reply:
Something for the maintainer call. We could certainly separate the developer documentation more from the end user documentation.